NtupleProducer/test/analyzer_LLR.py

```diff
@@ -11,7 +11,7 @@
 #YEAR   = 2016
```

Comment:
is everybody using this `analyzer_LLR.py` file or do we need to adapt `analyzer.py` as well?

Reply:
For what concerns py2 --> py3 analyser.py did not raise any error.